### ISSUE TITLE
Create a favicon for the map

### ISSUE THREAD

**Issue by frog23** [frog23/TomScottMap#51]
@Plushb9rry has suggested to add a favicon for the site and came up with two suggestions here: https://github.com/frog23/TomScottMap/pull/43#issuecomment-1179720204 , however both suggestions were not suitable as I explained here. https://github.com/frog23/TomScottMap/pull/50#issuecomment-1179729938

However it would still be nice to have a favicon for the page. Here are the thoughts that I already had regarding the topic. 

> My thought were something that combines an icon for a map with either a red marker (though this would be very similar to the old google maps favicon) or even a marker with a red t-shirt, but so far I had not the time to either create one myself or find suitable ones under a CC-Zero license that I can combine to create a proper one.

For that reason I created this issue, so that we have a central place for the discussion on what a favicon could look like and who could create it (I am not much of a graphics designer).

**Comment by Terminatortermi** [comment]
![favicon_tom](https://github.com/frog23/TomScottMap/assets/37415035/b2db6c5a-a427-4cd1-85be-9bc79517111e)
Did you have something like this in mind?

**Comment by frog23** [comment]
That looks awesome. Did you create it? Can I use this and if so, which license do you prefer?  Did you use any external materials/templates (I want to be sure to not use any materials with external licenses)?

I might need a couple of days to include this, depending on when I find the time for it. Thank you very much for this icon.

**Comment by Terminatortermi** [comment]
Yes I "created" it. It is basically just two images merged together.
I got the map from here: (https://www.flaticon.com/free-icon/map_854878). The license is free but I just realized that attribution is required. But this would be just a new link at the bottom of the page.
The shirt is a simple icon from FontAwesome (https://fontawesome.com/icons/shirt?f=classic&s=solid) which i downloaded in the offical Tom Scott red (#c02c38). FontAwesome also does not need a license and attribution is not required according to them (https://fontawesome.com/v4/license/)

**Comment by Maxklos** [comment]
I took the idea of @Terminatortermi but combined two [FontAwesome](https://fontawesome.com/license/free) icons. So there shouldn't be any problems with the license. But it doesn't look nearly as good.

![fav-icon](https://github.com/frog23/TomScottMap/assets/17532916/2931e321-fd32-4104-9cc7-9d11625acedd)
